connector/spanmetricsconnector/README.md

```diff
@@ -106,7 +106,9 @@ The following settings can be optionally configured:
   
   If no `default` is provided, this dimension will be **omitted** from the metric.
 - `exclude_dimensions`: the list of dimensions to be excluded from the default set of dimensions. Use to exclude unneeded data from metrics. 
-- `dimensions_cache_size` (default: `1000`): the size of cache for storing Dimensions to improve collectors memory usage. Must be a positive number. 
+- `dimensions_cache_size` (default: `1000`): the size of cache for storing Dimensions to improve collectors memory usage. Must be a positive number.
+- `instrumentation_scope`: a list of instrumentation scope names to include in the metrics. They are defined using `name` and `default` fields.
+- `exclude_instrumentation_scope`: a list of instrumentation scope names to exclude from the metrics.
```

Comment:
could this be a regex?

Reply:
and scope versions too?

Reply:
I tried to follow the same pattern than for dimensions. I'm not sure if a regex will make this easier to configure.